torch/quantization/convert_modules.py

```diff
@@ -0,0 +1,221 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch.nn as nn
+import torch.nn.quantized as nnq
+import torch
+from .observer import *
+
+def _forward_hook(self, input, output):
+    for module in self.children():
+        module.forward(output)
+
+def addObserver(myMod, qConfigDict, qConfigParent=None, prefix=''):
```

Comment:
what is our python naming scheme? I was under impression that it's names_with_underscores (please correct me if I'm wrong)

Reply:
Yeah I think we should use that, will change.